workspace.json

```diff
@@ -204,6 +204,21 @@
             ],
             "parallel": false
           }
+        },
+        "test-firestore": {
+          "builder": "@nrwl/jest:jest",
+          "options": {
+            "jestConfig": "apps/firebase-api/jest.config.js",
+            "passWithNoTests": true
+          }
+        },
+        "emulators-start-firestore": {
```

Comment:
Maybe find a way to run this as a prerequisite to the main `test` command. Ask Patrick for details.